apps/codeforafrica/src/payload/blocks/GetInTouch.js

```diff
@@ -0,0 +1,29 @@
+import link from "../fields/links/link";
+
+const GetInTouch = {
+  slug: "get-in-touch",
+  imageURL: "/images/cms/blocks/get_in_touch.jpg",
+  imageAltText: "Display Get In Touch Call to Action",
+  fields: [
+    {
+      name: "title",
+      label: "Title",
+      required: true,
+      type: "text",
+    },
+    {
+      name: "subtitle",
+      label: "Subtitle",
+      required: true,
+      type: "text",
+    },
+    {
+      name: "action",
+      label: "Action",
+      type: "group",
+      fields: [link()],
+    },
```

Comment:
```suggestion
    linkGroup({ overrides: { name: "action", label: "Action" } }),
```

I think we have `linkGroup` field for this.